components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/DefaultOIDCClaimsCallbackHandler.java

```diff
@@ -535,7 +536,11 @@ private Map<String, Object> getUserClaimsInOIDCDialect(String spTenantDomain,
         }
 
         List<String> requestedClaimUris = getRequestedClaimUris(requestClaimMappings);
-        Map<String, String> userClaims = getUserClaimsInLocalDialect(fullQualifiedUsername, realm, requestedClaimUris);
+        List<String> runtimeClaimsRemovedRequestedClaimUris =
+                requestedClaimUris.stream().filter(claim -> !claim.startsWith("http://wso2.org/claims/runtime/"))
```

Comment:
can't we just remove the claim?
`requestedClaimUris.remoteIf(claim -> !claim.startsWith("http://wso2.org/claims/runtime/")`

Reply:
fixed in [940bec1](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1931/commits/940bec12204f017305e1f3ba9f6f82f80c0bc9da)